.scalafmt.conf

```diff
@@ -2,4 +2,7 @@ version = "3.4.2"
 runner.dialect = scala212
 preset = Scala.js
 maxColumn = 120
-trailingCommas = always
+verticalMultiline.atDefnSite = true
+verticalMultiline.arityThreshold = 4
+trailingCommas = multiple
```

Comment:
This way we only add trailing commas when there's more than 1 element in the sequence (and they are across multiple lines).